### PR TITLE
PMM-11031 advanced-settings-UI

### PR DESCRIPTION
**Pull request by rasika-chivate**

### CODE REVIEW

File: docs/get-started/dbaas.md [hunk 1]
```diff
@@ -165,52 +159,42 @@ To create a DB cluster, do the following:
...
+    - *Database Version*
 
-    1. Enter a value for *Cluster name*. A cluster name:
-        - must begin with a lowercase letter;
-        - can comprise lowercase letters, numbers and dashes;
-        - must end with an alphanumeric character.
+4. Expand  *Advanced Settings* panel.
 
-    2. Select a cluster from the *Kubernetes Cluster* menu.
+    1. Select the number of nodes. (The lower limit is 3.)
```
Comment (beatahandzelova) [inline docs/get-started/dbaas.md:169]: We can create single node too. Limit is 1.

File: docs/get-started/dbaas.md [hunk 2]
```diff
@@ -165,52 +159,42 @@ To create a DB cluster, do the following:
...
 
-        *External Access* can't be granted for local Kubernetes clusters (e.g. minikube).
+        ![!](../_images/PMM_DBaaS_DB_Cluster_Advanced_DB_Config.png)
 
-    4. Select a preset for *Resources per Node*.
+    4. Enter the following on the *Network and Security* panel:
 
-        *Small*, *Medium* and *Large* are fixed preset values for *Memory*, *CPU*, and *Disk*.
+    - *Expose* - Select this checkbox to make this DB cluster available outside of the kubernetes cluster.
+    - *Internet Facing* - Select this checkbox if you want the cluster to be accessible on the internet.
```
Comment (beatahandzelova) [inline docs/get-started/dbaas.md:191]: Internet with capital "I".

**Comment by beatahandzelova** [comment]
@rasika-chivate there's a bug visible in these new screenshots https://jira.percona.com/browse/PMM-11369. Can you replace them so we don't see the $ sign please?

**Review by gen1us2k — APPROVED** [review]

**Review by denisok — CHANGES_REQUESTED** [review]
@solovevayaroslavna this screenshot for the statuses needs to be updated:
https://github.com/percona/pmm-doc/pull/958/files#diff-28552b691583f7b3f6d9e1db1e59975df317924b6ab9e43fc72782a18857466bR127
`../_images/PMM_DBaaS_Kubernetes_Cluster_Operator_Status.png`

@rasika-chivate Use helm list is broken:
https://github.com/percona/pmm-doc/pull/958/files#diff-28552b691583f7b3f6d9e1db1e59975df317924b6ab9e43fc72782a18857466bR35
in `## Add a Kubernetes cluster automatically` list starts with 1 and continuous with 1.

**Review by denisok — APPROVED** [review]

**Pull request merged**